src/services/ddev/ddev-service.ts

```diff
@@ -0,0 +1,101 @@
+import * as vscode from 'vscode';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+import { Sequelize } from 'sequelize';
+import { getConnectionFor } from '../sequelize-connector';
+
+const execAsync = promisify(exec);
+
+interface DdevConfig {
+  raw: {
+    database_type: string;
+    dbinfo: {
+      database_type: string;
+      database_version: string;
+      dbPort: string;
+      dbname: string;
+      host: string;
+      password: string;
+      published_port: number;
+      username: string;
+    };
+  };
+}
+
+/**
+ * Checks if DDEV is available in the current environment
+ * @returns Promise<boolean> indicating if DDEV is installed
+ */
+export async function isDdevAvailable(): Promise<boolean> {
+  try {
+    await execAsync('ddev --version');
+    return true;
+  } catch (error) {
+    console.log('DDEV is not available:', error);
+    return false;
+  }
+}
+
+/**
+ * Retrieves DDEV configuration by executing 'ddev describe -j' command
+ * @returns Promise<DdevConfig | undefined> The parsed DDEV configuration or undefined if command fails
+ */
+export async function getDdevConfig(): Promise<DdevConfig | undefined> {
+  try {
+    let path = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath
+    if (!path) {
+      throw new Error('No workspace folder found');
+    }
+
+    let folder = path.split('/').pop();
+    const { stdout } = await execAsync(`ddev describe -j ${folder}`, { cwd: path });
```

Comment:
There’s no guarantee that the folder name will match the DDEV project name.

I think you can simply check it without using `${folder}`.